articles/how-to-replicate-postgresql-database/index.md

```diff
@@ -0,0 +1,226 @@
+Replication is the act of reproducing or copying with something Several Relational Database Management Systems have different ways to achieve this. PostgreSQL being an open-source RDMS is what we are going to use in this article and show how replication is achieved.
+
+### Introduction
+
+For replication to be achieved in PostgreSQL there must be two servers that can be able to communicate with each other. This server will be identified as Master which is the Master server or the production server and the other one is the Slave server or Replica server or standby Server which will have a copy of master server data. The data is synchronized in such a way that if the Master server fails one can continue to run with the slave database without any problem.
+Replication is important to avoid Failover whereby the primary database fails and due to business continuity, there is a need for recovery. This recovery will depend on the replicated database created on Slave Server.
+
+### Why Replication
+
+1. Online Transaction Processing(OLTP) Performance- removing reporting query load from OLTP system improves both query time and transaction processing time
+2. Data migration- which comes about either through system deployment or change of database server hardware
+3. System Testing in Parallel- when upgrading a new system there need to make sure the new system works well with existing data hence need to test with production database copy before deployment
+4. Fault Tolerance- in case the main server fails the standby server can act as a server since the contained data  is the same
+
+For this to happen there must be communication between the two hosts or two servers that is through a network or internet as shown below.
+
+![LOCAL-EXAMPLE](/How-to-Replicate-PostgreSQL-database/local-network.png)
+
+![INTERNET-EXAMPLE](/How-to-Replicate-PostgreSQL-database/internet-network.png)
```

Comment:
In this image, you have made a typo, "Master Servaer"

Please correct it to "Master Server"